plugins/manual_classification/manual_classification_templates/manual_classification_template.html

```diff
@@ -1,25 +1,66 @@
 {% import 'forms.html' as forms %}
-<style type="text/css">
-    @import url("{{url_for('static', filename='forms.css')}}");
-</style>
-
-<h2>Manual Classification</h2>
-
-{% call forms.render_form() %}
-<!-- class name -->
-{{ forms.render_fields(schema, values=values, errors=errors) }}
-<div class="qhana-form" style="flex-direction: row; flex-wrap: wrap;">
-    <!-- select id's -->
-    {% for id in id_list %}
-    <div class="qhana-form-field" style="width: 3em; height: 2em; padding: 1em;">
-        <input type="checkbox" id={{id}} name={{id}}>
-        <label class="qhana-form-label" for="{{id}}">{{id}}</label>
+<html>
+
+<head>
+    <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='forms.css')}}" media="screen" />
+    <style type="text/css">
+        @import url("{{url_for('static', filename='forms.css')}}");
+
+        table {
+            font-family: arial, sans-serif;
```

Comment:
do not specify fonts if possible to allow for easier font overrides in the loaded micro frontend